src/components/UserProfile.jsx

```diff
@@ -1,15 +1,51 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import '../styles/index.scss';
 import ninja from '../assets/images/ninja.jpg';
+import { increment, decrement } from '../actions';
 
 const UserName = () => <h2>Forsetti</h2>;
 
 const Avatar = () => <img src={ninja} alt="ninja"/>;
 
-const Userprofile = () => 
-    <div>
-        <UserName />
-        <Avatar />,
-    </div>
+class Userprofile  extends Component{
```

Comment:
The class name should be the same with the filename

Reply:
Hi @paradisekelechi, both are named `UserProfile`. Is there something I may be missing?

Reply:
I get it now. Done